components/admin_console/brand_image_setting/brand_image_setting.jsx

```diff
@@ -62,86 +70,129 @@ export default class BrandImageSetting extends React.PureComponent {
                     brandImageTimestamp: Date.now(),
-                    status: UploadStatuses.COMPLETE,
                 });
             },
             (err) => {
                 this.setState({
                     error: err.message,
-                    status: UploadStatuses.DEFAULT,
                 });
+                this.props.onChange('saveNeeded', 'config');
             }
         );
     }
 
     render() {
-        let btnPrimaryClass = 'btn';
-        if (this.state.brandImage) {
-            btnPrimaryClass += ' btn-primary';
-        }
-
         let letbtnDefaultClass = 'btn';
         if (!this.props.disabled) {
             letbtnDefaultClass += ' btn-default';
         }
 
         let img = null;
+        let displayImgOverlay = false;
+        const IMGOVERLAY = (
+            <OverlayTrigger
+                delayShow={Constants.OVERLAY_TIME_DELAY}
+                placement='right'
+                overlay={(
+                    <Tooltip id='removeIcon'>
+                        <div aria-hidden={true}>
+                            <FormattedMessage
+                                id='admin.team.removeBrandImage'
+                                defaultMessage='Remove brand image'
+                            />
+                        </div>
+                    </Tooltip>
+                )}
+            >
+                <button
+                    className='remove-image__btn'
+                    onClick={() => {
+                        this.setState({deleteBrandImage: true, brandImage: null, brandImageExists: false});
+                        this.props.onChange('saveNeeded', 'config');
+                    }}
+                >
+                    <span aria-hidden={true}>{'×'}</span>
+                </button>
+            </OverlayTrigger>
+        );
         if (this.state.brandImage) {
             img = (
-                <img
-                    ref='image'
-                    className='brand-img'
-                    alt='brand image'
-                    src=''
-                />
+                <div className='remove-image__img margin-bottom x3'>
+                    <img
+                        ref='image'
+                        alt='brand image'
+                        src=''
+                    />
+                </div>
             );
         } else if (this.state.brandImageExists) {
             img = (
-                <img
-                    className='brand-img'
-                    alt='brand image'
-                    src={Client4.getBrandImageUrl(this.state.brandImageTimestamp)}
-                />
+                <div className='remove-image__img margin-bottom x3'>
+                    <img
+                        alt='brand image'
+                        src={Client4.getBrandImageUrl(this.state.brandImageTimestamp)}
+                    />
+                </div>
             );
+            displayImgOverlay = !this.props.disabled;
```

Comment:
Since we only use `IMGOVERLAY` in this one case, maybe we could move it so that we only define it here? We could do something like
```javascript
} else if (this.state.brandImageExists) {
    let overlay;
    if (!this.props.disabled) {
        overlay = (...);
    }

    img = (
        <div className='remove-image__img margin-bottom x3'>
            <img .../>
            {overlay}
        </div>
    );
} else {
```

Reply:
Resolved!